apps/communications/contacts/views/form/js/boot.js

```diff
@@ -0,0 +1,59 @@
+/* global LazyLoader, FormUI, Form */
+'use strict';
+
+
```

Comment:
nit: We should try to use jsdoc, here in this case we could simple notice this is the file that will be in charge of loading all the form dependencies. And can be modified here to use different set of ui view, or controller, etc.